manager_helpers/dask_manager_template.py

```diff
@@ -0,0 +1,106 @@
+"""
+Dask Distributed Manager Helper
+
+Conditions:
+- Dask Distributed and Dask Job Queue (dask_jobqueue in Conda/pip)
+- Manager running on the head node
+- SLURM manager
+
+For additional information about the Dask Job Queue, please visit this site:
+https://jobqueue.dask.org/en/latest/
+"""
+
+# Fractal Settings
+# Location of the Fractal Server you are connecting to
+FRACTAL_ADDRESS = "localhost:7777"
```

Comment:
We need username/password/update_frequency/verify here as well. Storing the password as plaintext makes me a bit nervous...